modules/apps/dynamic-data-mapping/dynamic-data-mapping-service/src/main/java/com/liferay/dynamic/data/mapping/verify/DDMServiceVerifyProcess.java

```diff
@@ -273,10 +273,11 @@ protected void verifyStructures() throws Exception {
 					catch (PortalException pe) {
 						_log.error(
 							String.format(
-								"Invalid data for DDM structure %d causes: " +
-									"{%s}",
-								ddmStructure.getStructureId(), pe.getMessage()),
-							pe);
+								"Invalid data for DDM structure %d causes " +
+									"exception",
+								ddmStructure.getStructureId()));
```

Comment:
Hi @SGM3, my goal here wasn't to make the log message more generic but to avoid redundancy. We are throwing the exception caught here and processing it here (https://github.com/liferay/liferay-portal/blob/master/modules/apps/portal/portal-verify-extender/src/main/java/com/liferay/portal/verify/extender/internal/osgi/commands/VerifyProcessTrackerOSGiCommands.java#L295) where the error is logged in detail. But I thought it was still relevant to point out the DDM structure ID that causes the failure. (Ideally we'd simply process the exception in one place or the other but I didn't see a clean way to do that without preventing relevant information from being logged.)